Week1/exercise_1.js

```diff
@@ -0,0 +1,76 @@
+const mysql = require('mysql');
+
+const connection = mysql.createConnection({
+    host: 'localhost',
+    user: 'hyfuser',
+    password: 'hyfpassword',
+    multipleStatements: true
+});
+
+
+/*Connect to SQL server*/
+connection.connect(err => {
+    if (err) {
+        return console.error('Connection error: ' + err.stack);
```

Comment:
Would be nice to know what the error message says too rather than the stack trace